assets/queries/common/passwords_and_secrets_in_infrastructure_code/query.rego

```diff
@@ -0,0 +1,290 @@
+package Cx
```

Comment:
run `opa fmt -w assets/queries/common/passwords_and_secrets_in_infrastructure_code/query.rego` to properly format your file